docs/DEPLOYING.md

```diff
@@ -103,6 +103,13 @@ using the `--source` argument to point to `janus/db` and providing database
 connection information in any of the ways supported by `sqlx` (see its
 documentation).
 
+For simple or experimental deployments where the complexity of `sqlx` is not
+warranted, it is possible to create a single schema file by concatenating the
+`.up.sql` scripts, in order, and applying this schema to the database. Note that
```

Comment:
Since my last comment, I realized that `check_schema_version: false` will have to be set when using this method. I'm working on a separate change to add `check_schema_version: true` in the sample files, after which the below sentence will be self-explanatory.

```suggestion
`.up.sql` scripts, in order, and applying this schema to the database.
When using this technique, `check_schema_version: false` must be set in each configuration file.
Note that
```